### DISCUSSION TITLE
How to use models: Cmdr2's Stable Diffusion UI

### DISCUSSION THREAD

**Discussion by JeLuF** [civitai/civitai#12] [Q&A] [answered]
The instructions for Cmdr2's Stable Diffusion UI:

# [Stable Diffusion UI v2](https://stable-diffusion-ui.github.io/)

1. Download the custom model in Checkpoint format (`.ckpt`)
2. Place the model file inside the `models\stable-diffusion` directory of your installation directory (e.g. `C:\stable-diffusion-ui\models\stable-diffusion`)
3. Reload the web page to update the model list
4. Select the custom model from the _Model_ list in the Image Settings section
5. Use the trained keyword in a prompt (listed on the custom model's page)

**Comment by JustMaier** [comment, marked as answer]
Thanks for providing these. This has been added to the Wiki 👍
